src/components/AccountNamesTableCondensed.vue

```diff
@@ -55,6 +70,7 @@
 import AppLink from '@/components/AppLink'
 import ValueHashEllipsed from '@/components/ValueHashEllipsed'
 import DatetimeLabel from '@/components/DatetimeLabel'
+import { accountHints } from '~/utils/hints/accountHints'
```

Comment:
app-tooltip component is missing

Reply:
Sometimes I forgot them, sometimes I didn't import it intentionally. I wanted to see how the autoimport works. Seems like for months there has been no issue with it and no one noticed. It's the thing I wanted to discuss on last conference, but there was very little time to cover all topics. I would like to give it a try. For the same reason we use magical auto router. I see imports as a technical boilerplate given the framework we are using. Could we discuss it here? https://github.com/aeternity/aescan/issues/112

Reply:
Yes, currently, the autoimport of components, pages, and stores is activated and we don't need to use it. I'm saying it only because we agreed to still import files until #112 is approved by everybody and resolved.

Reply:
For now I added the imports and initiated discussion